Common/src/main/resources/assets/biomeswevegone/lang/ru_ru.json

```
@@ -1,18 +1,26 @@
 {
+  "advancements.biomeswevegone.adventure.forgotten_fae.description": "Найдите музыкальный диск Клуб Пикси",
+  "advancements.biomeswevegone.adventure.forgotten_fae.title": "Забытые феи",
   "advancements.biomeswevegone.adventure.inside_quicksand.description": "Погрузитесь в зыбучий песок.",
```

При отсутствии знаков препинания "!", "?" и "..." в конце описаний достижений не должно быть точек
```suggestion
  "advancements.biomeswevegone.adventure.inside_quicksand.description": "Погрузитесь в зыбучий песок",
```